addons/variable-manager/style.css

```diff
@@ -0,0 +1,47 @@
+.sa-var-manager {
+  display: block;
+  padding: 8px;
+}
+
+.sa-var-manager.freeze > * > * > * > * > * {
+  opacity: 0.5;
+}
+
+.sa-var-manager.freeze > * > * > * > * > input:focus {
```

Comment:
wouldn't it be easier to just do
`.sa-var-manager.freeze input:focus`

Reply:
oh yeah